package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "metamocha",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "chai": "^4.0.2",
+    "deepmerge": "^1.4.4",
+    "fs": "0.0.1-security",
```

Comment:
whats up with this dependency? `fs` is a built-in node module, was there a particular issue you were working around?